install/linux/usr/share/odemis/hwtest/clspots-sem-diagnostic.odm.yaml

```diff
@@ -0,0 +1,86 @@
+MultiBeamSEM: {
+  class: Microscope,
+  role: mbsem,
+  children: ["SEM", "EBeam Focus", "EBeam Stage", "EBeam control",
+             "DiagnosticCam", "DiagnosticCam Focus", "Sample Stage"],
+}
+
+# Axes: X is horizontal on screen (going left->right), physical: far->close when looking at the door
+#       Y is vertical on screen (going bottom->top), physical: left->right when looking at the door
+"DiagnosticCam": {
+  class: ueye.Camera,
+  role: diagnostic-ccd,
+  init: {
+    device: null,
+  },
```

Comment:
We might need to check with Andries again. I think they inverted now the ueye in the yaml to swap the coordinate system from left handed to right handed.